formats/message_queue_base.json

```diff
@@ -0,0 +1,148 @@
+  },
+  "allOf": [
+    {
+      "additionalProperties": false,
+      "required": [
+        "update_type",
+        "analytics_identifier",
+        "content_id",
+        "description",
+        "details",
+        "document_type",
+        "first_published_at",
+        "need_ids",
+        "phase",
+        "public_updated_at",
+        "publishing_app",
+        "redirects",
+        "rendering_app",
+        "routes",
+        "schema_name",
+        "title",
+        "base_path",
+        "locale",
+        "format"
+      ],
+      "properties": {
+        "update_type": {
+          "enum": [ "major", "minor", "republish", "links" ]
+        },
+        "content_id": {
+          "$ref": "#/definitions/guid"
+        },
+        "analytics_identifier": {
+          "$ref": "#/definitions/analytics_identifier"
+        },
+        "links": {
+          "$ref": "#/definitions/links"
+        },
+        "expanded_links": {
+          "type": "object"
+        },
+        "base_path": {
+          "$ref" : "#/definitions/absolute_path"
+        },
+        "title": {
+          "type": "string"
+        },
+        "description": {
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "public_updated_at": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "first_published_at": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "publishing_app": {
+          "type": "string"
+        },
+        "rendering_app": {
+          "type": "string"
+        },
+        "locale": {
+          "type": "string"
+        },
+        "need_ids": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "title": "This is intended to be used by front end applications to present pages to users to show how effectively users' needs are being met"
+          },
+          "title": "Identifiers of user needs"
+        },
+        "routes": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/route"
+          }
+        },
+        "redirects": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/redirect_route"
+          }
+        },
+        "access_limited": {
+          "type": "object",
+          "additionalProperties": false,
+          "required": [ "users" ],
+          "properties": {
+            "users": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "phase": {
+          "description": "The service design phase of this content item - https://www.gov.uk/service-manual/phases",
+          "type": "string",
+          "enum": [ "alpha", "beta", "live" ]
+        },
+        "details": {
+          "type": "object"
```

Comment:
Yes, I realised [as I looked at the generator](https://github.com/alphagov/govuk-content-schemas/pull/333/files/df2cba8d98c4dd4234014c7b645f3251edc71414#diff-07d491c55fca3fc137b1fcc036090840R41) that it was academic what the type was, ignore that. :+1: